rainier-core/src/main/scala/com/stripe/rainier/core/Continuous.scala

```diff
@@ -8,16 +8,26 @@ import scala.annotation.tailrec
   * A Continuous Distribution, with method `param` allowing conversion to a RandomVariable
   */
 trait Continuous extends Distribution[Double] { self =>
-  def param: RandomVariable[Real]
-
   def logDensity(t: Double): Real =
     realLogDensity(Real(t))
 
+  private[rainier] val support: Support
+
   def scale(a: Real): Continuous = Scale(a).transform(this)
   def translate(b: Real): Continuous = Translate(b).transform(this)
   def exp: Continuous = Exp.transform(this)
 
   private[rainier] def realLogDensity(real: Real): Real
+
+  def param: RandomVariable[Real] = {
```

Comment:
nit: rather than having this in `Continuous` and overriding it in `Injection`, can we move it down to some private `BaseContinuous` trait that everything other than `Injection` can inherit from? I prefer to avoid overrides if at all possible.

Reply:
We could also move `support` down there for now I think